From 5fd2ca1d82570c1053720b379b3bbaf17c9ba073 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan <ra5478@freescale.com>
Date: Sat, 13 Oct 2012 18:38:25 -0500
Subject: [PATCH 2142/2463] ENGR00229464 MX6SL-Update the SOC voltages based
 on datasheet

Update the VDDARM and VDDSOC voltages based on IMX6SLCEC_Rev0
datasheet.
As the voltages for ARM @ 198MHz and ARM @ 396MHz are the same
remove the 198MHz working point.

Signed-off-by: Ranjani Vaidyanathan <ra5478@freescale.com>
---
 arch/arm/mach-mx6/cpu_op-mx6.c |   38 +++++++++++---------------------------
 1 files changed, 11 insertions(+), 27 deletions(-)

diff --git a/arch/arm/mach-mx6/cpu_op-mx6.c b/arch/arm/mach-mx6/cpu_op-mx6.c
index 99336d8..e6ef026 100644
--- a/arch/arm/mach-mx6/cpu_op-mx6.c
+++ b/arch/arm/mach-mx6/cpu_op-mx6.c
@@ -217,32 +217,24 @@ static struct cpu_op mx6sl_cpu_op_1G[] = {
 	 .pll_rate = 996000000,
 	 .cpu_rate = 996000000,
 	 .cpu_podf = 0,
-	 .pu_voltage = 1225000,
-	 .soc_voltage = 1225000,
-	 .cpu_voltage = 1275000,},
+	 .pu_voltage = 1200000,
+	 .soc_voltage = 1200000,
+	 .cpu_voltage = 1250000,},
 	{
 	 .pll_rate = 792000000,
 	 .cpu_rate = 792000000,
 	 .cpu_podf = 0,
 	 .pu_voltage = 1150000,
 	 .soc_voltage = 1150000,
-	 .cpu_voltage = 1200000,},
+	 .cpu_voltage = 1150000,},
 	{
 	 .pll_rate = 396000000,
 	 .pll_lpm_rate = 792000000,
 	 .cpu_rate = 396000000,
 	 .cpu_podf = 0,
-	 .pu_voltage = 1050000,
-	 .soc_voltage = 1050000,
-	 .cpu_voltage = 1100000,},
-	 {
-	  .pll_rate = 396000000,
-	 .pll_lpm_rate = 792000000,
-	  .cpu_rate = 198000000,
-	  .cpu_podf = 1,
-	  .pu_voltage = 1050000,
-	  .soc_voltage = 1050000,
-	  .cpu_voltage = 1050000,},
+	 .pu_voltage = 1150000,
+	 .soc_voltage = 1150000,
+	 .cpu_voltage = 950000,},
 };
 
 static struct cpu_op mx6sl_cpu_op[] = {
@@ -252,23 +244,15 @@ static struct cpu_op mx6sl_cpu_op[] = {
 	 .cpu_podf = 0,
 	 .pu_voltage = 1150000,
 	 .soc_voltage = 1150000,
-	 .cpu_voltage = 1200000,},
+	 .cpu_voltage = 1150000,},
 	 {
 	  .pll_rate = 396000000,
 	 .pll_lpm_rate = 792000000,
 	  .cpu_rate = 396000000,
 	  .cpu_podf = 0,
-	 .pu_voltage = 1050000,
-	 .soc_voltage = 1050000,
-	 .cpu_voltage = 1100000,},
-	{
-	 .pll_rate = 396000000,
-	 .pll_lpm_rate = 792000000,
-	 .cpu_rate = 198000000,
-	 .cpu_podf = 1,
-	  .pu_voltage = 1050000,
-	  .soc_voltage = 1050000,
-	  .cpu_voltage = 1050000,},
+	 .pu_voltage = 1150000,
+	 .soc_voltage = 1150000,
+	 .cpu_voltage = 950000,},
 };
 
 static struct dvfs_op dvfs_core_setpoint_1_2G[] = {
-- 
1.7.7.4

